Responsive design

Lucas-Erkana opened the pull request:
✅Use media queries to make the website responsive.

Lucas-Erkana committed:
Change media from max-width to min-width 768

Lucas-Erkana committed:
Remove background attachment in media headline

Lucas-Erkana committed:
Remove description class and move it to p tag

Lucas-Erkana committed:
Add new svg files for background and change url for headline

Lucas-Erkana committed:
Add missing projects such as Availabilty, Areto , Uber page and Facebook page

Lucas-Erkana committed:
Add grid-container around second_project and sixth_project

Lucas-Erkana committed:
Add add grid styling to the grid-container

Lucas-Erkana committed:
Add grid-container around first_project and eighth_project

Lucas-Erkana committed:
Add grid-container around ninth_project and tenth_project

Lucas-Erkana committed:
Add grid-container around About section and foooter

Lucas-Erkana committed:
Clean style todeal with all the spaces and indentation plus duplicates

MaganaAsiati reviewed (changes requested):
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @Lucash2022 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close-gif-4](https://user-images.githubusercontent.com/95297251/191952218-b7185270-c02b-4e4d-89af-eae77fcd8bea.gif)

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

- Kindly make sure your design sticks to the Figma design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :+1: 

- Kindly the hamburger menu must not be present in the desktop version, look at the design of the nav bar in the desktop version to the [ Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)  and make sure it looks exactly the same  :+1: 
- You did a great job implementing the card section but on my side, there are not visible in the desktop version please fix that and make sure you stick the design to the Figma design 
 :+1: 
- Kindly make sure the contact section and about are responsible in the desktop version, check the design to  Figma and make sure your design sick to the desktop version in Figma design   :+1: 


## your design
 
### Home page 
![image](https://user-images.githubusercontent.com/95297251/199759500-b7697559-1301-4a0a-83e5-6b79605c3915.png)

## Figma design

### Home page
 
![image](https://user-images.githubusercontent.com/95297251/199760046-be9bf01f-d8e6-4537-a320-c4d86f0f7729.png)

### cards section 

![image](https://user-images.githubusercontent.com/95297251/199764900-3efc591e-c217-4e59-8b8b-61ac8fa25da6.png)

### about section and contact section 

![image](https://user-images.githubusercontent.com/95297251/199765265-ba0a1619-5af0-49b8-b730-7eabee3bb16d.png)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [x] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻
- [x] kindly  Add a meaningful description to your pull request in the future which gives more information about what you implement in this milestone :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

Lucas-Erkana committed:
Change the structure of the index.html, add 2 sections, place projects inside sections

Lucas-Erkana committed:
Add Availability.svg file to svg_files folder

Lucas-Erkana committed:
Add Explore.svg and Explore.png file to svg_files folder

Lucas-Erkana committed:
Add Professional.svg and Professional.png file to svg_files folder

Lucas-Erkana committed:
Add Professional.png file to svg_files folder

Lucas-Erkana committed:
Add Nature.svg and Nature.png file to svg_files folder

Lucas-Erkana committed:
Update MIT file

Lucas-Erkana committed:
Add margin 0, padding 0, box-sizing and background color

Lucas-Erkana committed:
Add styles(font-style,font-weight,size.line-height,color,list-style,width,display to .logo selector

Lucas-Erkana committed:
Apply text decoration to all anchor selectors, plus color to inherit, plus cursor to auto

Lucas-Erkana committed:
Set margin and PAdding to zero, and box-sizing to border box for all content

Lucas-Erkana committed:
Apply display, flex direction, align-item, width, jusity-content, background-color and padding properties to nav selector

Lucas-Erkana committed:
Apply display, flex direction, gap and list-style propeties to ul selector

Lucas-Erkana committed:
Adjust flex layout

Lucas-Erkana committed:
Adjust flex item to start

Lucas-Erkana committed:
Adust width to 100% to nav selector

Lucas-Erkana committed:
Apply white background color to nav selector

Lucas-Erkana committed:
Adjust flex layout

Lucas-Erkana committed:
Add background image to healine selector

Lucas-Erkana committed:
Adjust flext layout to introduction selector

Lucas-Erkana committed:
Apply fonts to h1 selector

Lucas-Erkana committed:
Aoply font to description selector

Lucas-Erkana committed:
Appply styles project_section selectors

Lucas-Erkana committed:
Add media query for max-width 600px for mobile version

Lucas-Erkana committed:
Apply fonts to list and list items selector

Lucas-Erkana committed:
Apply styles to About section

Lucas-Erkana committed:
Enclose language class within About section

Lucas-Erkana committed:
Adjust flex layout to conatiner and languages selector

Lucas-Erkana committed:
Fix identation and space errors in css

Lucas-Erkana committed:
Fix link to Professional.png

Lucas-Erkana committed:
Add Availability, Explore selectors to media query 600px

Lucas-Erkana committed:
Add Profession png and svg files to svg_files folder

Lucas-Erkana committed:
Add style to contact_paragraph selectore

Lucas-Erkana committed:
Add media query for screen width 768

Lucas-Erkana committed:
Merge branch 'main' into RVD

Felix45 reviewed (changes requested):
Hi @Lucash2022 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

 ## Highlights :heavy_check_mark: 
- [x] All linter checks are passing
- [x] Good descriptive commit messages


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lucas-Erkana reviewed (commented):
Great work, But you need to ,make some changes left by  the reviewer

Lucas-Erkana committed:
Fix background image

@Felix45 Fixed the background image issue

Lucas-Erkana committed:
Upload new image 

The image is called Availability)small.svg
The image will be used for the mobile version

Lucas-Erkana committed:
Update index.html

-Add muli_project class
-Add tonic image for mobile version
-Add tonic_project for mobile version

Lucas-Erkana committed:
Update style.css to fix issues 

-Add display none for items that don't need to be displayed in the mobile version(0-600px)
-Add display none for items that don't need to be displayed in the desktop version(768px above)
-Reduce the size of Tonic and Availability images to a smaller size for mobile version
-Improve the navbar style

Lucas-Erkana committed:
Fix styles.css errors

-Removed space before "{"
-Fix "❌ Expected empty line before rule"
-Fix "❌ Expected single space before "{""
-Fix  "❌ Unexpected whitespace at end of line"

Lucas-Erkana committed:
Increase the size of the images

-Increase the size of Availability, Tonic and Professional to 700px

Lucas-Erkana committed:
Fix button issue

-Set the margin of the form_button to auto

Lucas-Erkana committed:
Fix background on contact form

Lucas-Erkana committed:
Fix background on contact form to cover

mgmediaweb reviewed (changes requested):
Hi @Lucash2022 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![almost3](https://user-images.githubusercontent.com/41912912/200100465-6480968c-0197-4c6c-9702-877149420a41.gif)

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Lucas-Erkana committed:
Update README file

-Added a link to a live demo of thsite

Lucas-Erkana committed:
Adjust parapgraph

-Added a form_paragraph to the media query for 768px
-Added align-self to center

Debas-31 reviewed (approved):
> ## Hi @Lucash2022  :wave, 

# Status: Approved :green_circle: 

Great work on making the changes requested by a previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
<img src="https://media1.giphy.com/media/fYlennPcKut8gjKOZD/giphy.gif"/>

## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 

## Optional suggestions
[OPTIONAL] - Kindly work on the responsiveness in screen size 768 px to 1440 px. Some of your designs did not align properly :eagle: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
